pkg/epp/scheduling/framework/plugins/scorer/lora_affinity.go

```diff
@@ -27,7 +27,7 @@ import (
 
 const (
 	DefaultLoraAffinityScorerWeight = 1
-	LoraAffinityScorerType          = "lora-affinity"
```

Comment:
perfect 🙏🏼
if we’re already here, can you remove the DefaultLoraAffinityScorer? 

those are not used anymore